src/utils/telemetry/index.js

```diff
@@ -88,70 +69,39 @@ const track = async function (eventName, payload) {
     return false
   }
 
-  const defaultProperties = {
-    // cliId: cliId
-  }
-
-  delete properties.force
-
   const defaultData = {
     event: eventName,
     userId,
     anonymousId: cliId,
-    properties: { ...defaultProperties, ...properties },
+    properties: payload,
   }
 
   return send('track', defaultData)
 }
 
 const identify = async function (payload) {
-  const data = payload || {}
```

Comment:
`identify` is only called once in https://github.com/netlify/cli/blob/b13d6d07afdfe4b932cb115daa161abf93fee12f/src/utils/command.js#L307 so `payload` is always defined.